pkg/spireentry/reconciler.go

```diff
@@ -208,6 +210,16 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 	}
 }
 
+func isNamespaceIgnored(ignoredNamespaces []*regexp.Regexp, namespace string) bool {
```

Comment:
Sorry i missed this before, can you move this function to the end of the file? Its not part of the struct.